frontend/coprs_frontend/commands/clean_expired_projects.py

```diff
@@ -11,6 +11,5 @@ def clean_expired_projects():
     Clean all the expired temporary projects.  This command is meant to be
     executed by cron.
     """
-
-    with db_session_scope():
-        ComplexLogic.delete_expired_projects()
+    while ComplexLogic.delete_expired_projects(limit=100):
```

Comment:
do you want to use the limit here?

Reply:
Nah, isn't this the reason why the script finished in 45 minutes?

Reply:
Yeah, I think we want the limit here. We can change the value though.

Reply:
ah, I missed the `s/with/while/` change, it seems OK then!  But don't we want to use a much smaller limit, like 5 :-)?